hazelcast/pom.xml

```diff
@@ -315,6 +315,29 @@
     </build>
 
     <dependencies>
+        <!-- needed for thread affinity -->
+        <dependency>
+            <groupId>net.openhft</groupId>
+            <artifactId>affinity</artifactId>
+            <version>3.2.2</version>
+            <scope>provided</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.25</version>
+            <scope>provided</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.5.8</version>
+            <scope>provided</scope>
+            <optional>true</optional>
+        </dependency>
+
```

Comment:
I think later for a product feature we should get rid of these dependencies. Ideally, we should have no added dependency if thread affinity becomes a prod feature. We can bundle a statically linked native lib in our jar - just like in EE - that only provides the glue code between the underlying syscalls and our Java code. If it would be done against libnuma - that also supports setting affinity -, we would gain additional features such as discovering the NUMA layout of the machine.

Reply:
Agree. I already added this to the TDD.